File: parent327/pinyougou-web-manager/src/main/webapp/js/service/orderQueryService.js
// 定义服务层:
app.service("orderQueryService",function($http){

    this.findOne=function(orderId){
        return $http.get("../orderQuery/findOne.do?id="+orderId);
    }
    this.search = function(page,rows,searchEntity){
        return $http.post("../orderQuery/search.do?pageNum="+page+"&pageSize="+rows,searchEntity);
    }
});

File: parent327/pinyougou-interface/src/main/java/cn/itcast/core/service/OrderQueryService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.order.Order;
import cn.itcast.core.pojo.seckill.SeckillOrder;
import entity.PageResult;

public interface OrderQueryService {
    PageResult search(Integer pageNum, Integer pageSize, Order order);

    Order findOne(Long id);
}
